### ISSUE TITLE
No PRs or Isuues are shown after update to 1.2.0 

### ISSUE THREAD

**Issue by kjgarza** [steamclock/Quests#2]
Quests 1.2.0 not working on OSx big sur 11.0.1. It does no get the issues and PRs can you help me?

there is only spinner but the list never appears/

```
2021-01-05 12:02:14.052 [Quests] > Quests Version: 1.2.0 Build: 202012221000 PID: 34633
2021-01-05 12:02:14.052 [Quests] > XCGLogger Version: 6.1.0 - Level: Debug
2021-01-05 12:02:14.052 [Quests] > XCGLogger writing log to: file:///Users/kristian/Library/Containers/com.steamclock.quests/Data/Documents/quests_debug.txt

```

**Comment by kjgarza** [comment]
it just stays like this forever:

![image](https://user-images.githubusercontent.com/1092861/103633148-9f5de280-4f45-11eb-9eed-ac637e519de5.png)

Anybody with a similar problem?

**Comment by kjgarza** [comment]
fixed in 1.2.1
